pkg/controllers/monitoring-stack/controller.go

```diff
@@ -145,14 +131,10 @@ func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	if requeue, err := r.createGrafanaDSWatch(ctx); err != nil {
-		return ctrl.Result{}, err
-	} else if requeue {
-		return ctrl.Result{RequeueAfter: 5 * time.Second}, nil
-	}
 
 	if !ms.ObjectMeta.DeletionTimestamp.IsZero() {
```

Comment:
Do you see this happen? IIUC, this path shouldn't be hit since we don't have finalizers set anymore.